ioservice/ut/bulkio_client.c

```diff
@@ -483,15 +483,19 @@ static void bulkclient_test(void)
 
 int bulkio_client_ut_init( void )
 {
-	m0_fi_enable("libfab_target_notify", "lf_dummy_msg_snd");
-	m0_fi_enable("libfab_buf_done", "lf_dummy_msg_rcv");
+	if (USE_LIBFAB) {
+		m0_fi_enable("libfab_target_notify", "lf_dummy_msg_snd");
+		m0_fi_enable("libfab_dummy_msg_rcv_chk", "lf_dummy_msg_rcv");
+	}
```

Comment:
In case of RDMA reads, we need to send an extra message to notify the target about the completion of RDMA read operation. This can be skipped as the completion of passive buffer can be taken care of by the RPC layer after receiving reply of fop. But this dummy message is still required for net-test and some other UTs which do not use the RPC layer. Hence this is enabled/disabled selectively using fault injection mechanism.